storage/mysql/map_storage.go

```diff
@@ -213,6 +199,21 @@ func (m *mapTreeTX) Get(ctx context.Context, revision int64, indexes [][]byte) (
 	if len(indexes) == 0 {
 		return []*trillian.MapLeaf{}, nil
 	}
+	selectMapLeafSQL := `
```

Comment:
This should probably still be a const unless you had some reason to change it to a var?
(`const selectMapLeafSQL = ...`)